web_programming/get_top_billioners.py

```diff
@@ -0,0 +1,41 @@
+from datetime import date, datetime
+
+import requests
+import tabulate
+
+API_URL = (
+    "https://www.forbes.com/forbesapi/person/rtb/0/-estWorthPrev/true.json"
+    "?fields=personName,gender,source,countryOfCitizenship,"
+    "birthDate,finalWorth&limit=10"
+)
+
+
+def real_time_billionaires() -> str:
+    """Get top 10 realtime billionaires using forbes API
+    Returns:
+    Top 10 realtime billionaires date in tabulated string.
```

Comment:
```suggestion

    Returns the Top 10 realtime billionaires date in tabulated string.
```

Reply:
This is a comment syntax that some editors (PyCharm, I think) prefer so we should leave it as is.

Reply:
Yeah, this was automatically done by PyCharm. However, I've modified it a bit. c85adac